scripts/deploy/src/install.js

```diff
@@ -0,0 +1,163 @@
+const fs = require('fs');
+const { execSync } = require('child_process');
+const axios = require('axios');
```

Comment:
Unfortunately axios is not part of our stack, and will probably never be. Please use `node-fetch` instead. 
Discussion: https://github.com/medic/cht-core/issues/6250

Reply:
Consider native [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) and setting a minimum node version, its API wasn't considered stable until node 21, but it has existed since 18 and I can't think of any breaking changes that would affect this. It's fine to use node-fetch if you don't want to verify compatibility

Reply:
Done.